### PR TITLE
Outlanders update

### PR DESCRIPTION
**Pull request by DarkoniusXNG**

### CODE REVIEW

File: game/scripts/npc/abilities/drow_ranger_multishot.txt [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    // Casting
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilityCastPoint"				"0.0"
+    "AbilityChannelTime"			"1.75"
+    "AbilityCastAnimation"		"ACT_DOTA_CHANNEL_ABILITY_3"
+
+    // Time
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilityCooldown"				"32 28 24 20 16 16"
```
Comment (POCKETMONEY99) [inline game/scripts/npc/abilities/drow_ranger_multishot.txt]: I did all of 7.23b has been done except for this.

**Comment by chrisinajar** [comment]
how can i help

**Comment by POCKETMONEY99** [comment]
Level cap needs adjusting. I was messing around in dota tools and I couldn't level up all my skills to max level on spirit breaker. I think the easiest solution is to increase the level cap to 55.

**Comment by DarkoniusXNG** [comment]
Level cap will stay 50. Valve fucked up cheats (-lvlup for example), its not a big deal.

**Comment by POCKETMONEY99** [comment]
> 
> 
> Level cap will stay 50. Valve fucked up cheats (-lvlup for example), its not a big deal.

ok. as long as you can level all your skills by level 50 its fine

**Comment by chrisinajar** [comment]
We've always had more ability points than we use, this is not a regression or new problem, we instead already override the HUD and use action filters to prevent you from leveling skills up early.

**Pull request merged**